package.json

```diff
@@ -29,20 +29,20 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "aws-sdk": "^2.559.0",
-    "bluebird": "^3.7.1",
+    "aws-sdk": "^2.810.0",
+    "bluebird": "^3.7.2",
```

Comment:
👍 Though probably could drop this out in a future version. I believe everything this package is using from bluebird is available from node directly now.